spec/stat_generator_spec.rb

```diff
@@ -24,6 +24,66 @@
     end
   end
 
+  describe '#checks highest total score and lowest total score' do
+    it 'Calculates the highest total score for all games' do
+      expect(@stat_generator.highest_total_score).to eq(11)
+      # @game2 = Game.new() try to prove that 
+      expect(@stat_generator.highest_total_score).to_not eq(1)
+    end
+
+    it 'Calculates the lowest total score of all games' do 
+      expect(@stat_generator.lowest_total_score).to eq(0)
+      expect(@stat_generator.lowest_total_score).to_not eq(9)
+
+    end
+    
+  end
+
+  describe '#percent wins at home ' do
+    it 'Sees the percent won at home ' do
+      
+      expect(@stat_generator.count_of_games).to_not eq(9)
+      expect(@stat_generator.count_of_games).to eq(7441)
+
+      expect(@stat_generator.total_home_wins).to_not eq(98)
+      expect(@stat_generator.total_home_wins).to eq(3237)
+
+      expect(@stat_generator.percentage_home_wins).to eq(0.44)
+      expect(@stat_generator.percentage_home_wins).to_not eq(3.09)
+    end
+  end
+  
+  describe '#percent wins away ' do
+  it 'Sees the percent won away ' do
+    
+    expect(@stat_generator.count_of_games).to_not eq(400)
+    expect(@stat_generator.count_of_games).to eq(7441)
+    
+    expect(@stat_generator.total_away_wins).to_not eq(40)
+    expect(@stat_generator.total_away_wins).to eq(2687)
+    
+    expect(@stat_generator.percentage_away_wins).to eq(0.36)
+    expect(@stat_generator.percentage_away_wins).to_not eq(3.09)
+    end
+  end
+  
+  describe '#percent ties ' do
+    it 'Sees the percent of how many games they had a tie' do
+      
+      expect(@stat_generator.count_of_games).to_not eq(292)
+      expect(@stat_generator.count_of_games).to eq(7441)
+      
+      expect(@stat_generator.total_ties).to_not eq(33)
+      expect(@stat_generator.total_ties).to eq(1517)
+      
+      expect(@stat_generator.percentage_ties).to eq(0.2)
+      expect(@stat_generator.percentage_ties).to_not eq(4.04)
+    end
+  end
+
+
+
+
```

Comment:
Since we have the expect().to eq(n) assertions we don't need the expect().to_not eq(n) assertions. This is because if the return value is a specific integer, it is not possible for it to equal another integer so these tests are redundant.